docs/en/styling-and-theming/supplemental.md

```diff
@@ -141,229 +151,148 @@ However, careful use of this feature can be helpful in large applications. For e
 
 ```css
 .root {
-	composes: buttonArchetype from '../common/ButtonArchetype.m.css';
+	composes: buttonBase from '../common/ButtonBase.m.css';
 	background-color: blue;
 }
 ```
 
-# Working with themes
-
-## Widget theme keys
-
-Dojo's theming framework uses the concept of a 'widget theme key' to connect style overrides to the corresponding widget that the styles are intended for. Style overrides are usually [specified in a theme](#writing-a-theme), but can also be passed [directly via `ThemedMixin`'s `classes` override property](#passing-extra-classes-to-widgets), if required.
-
-The theme key for a given widget is determined as:
-
-    {package-name}/{widget-css-module-name}
-
-where `package-name` is the value of the `name` property within the project's `package.json`, and `widget-css-module-name` is the filename of the primary CSS module used for the widget (_without_ the `.m.css` extension).
-
-### Theme key example
-
-For a given project:
-
-> package.json
-
-```json
-{
-	"name": "my-app"
-}
-```
-
-When [following widget CSS module naming conventions](#structural-widget-styling), a given widget such as `src/widgets/MyWidget.ts` will use a corresponding CSS module name similar to `src/styles/MyWidget.m.css`. The theme key for `MyWidget` is therefore:
-
-    my-app/MyWidget
-
-Here, the name of the widget is the same as the the name of its CSS module file, but developers should be careful not to mistake the widget's theme key as representing the widget's TypeScript class name.
+## Dojo styling best-practices
 
-For a second widget that does not follow CSS module naming conventions, such as `src/widgets/BespokeWidget.ts` that uses a corresponding CSS module such as `src/styles/BespokeStyleSheet.m.css`, its widget theme key would instead be:
+As styles in a Dojo application are mostly scoped to individual widgets, there is little need for complex selector targeting. Style application in Dojo should be as simple as possible - developers can achieve this by following a few simple recommendations:
 
-    my-app/BespokeStyleSheet
+-   Maintain encapsulated widget styling
+    -   Only refer to classes relevant for the corresponding widget within a given CSS module
```

Comment:
not sure this is entirely true, something along the lines of single css module per concern with the exception of intended shared modules, ie an app wide typography module